cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -125,3 +131,9 @@ def get_all_demultiplexed_flow_cell_dirs(self) -> list[Path]:
                 LOG.debug(f"Found directory {flow_cell_dir}")
                 demultiplex_flow_cells.append(flow_cell_dir)
         return demultiplex_flow_cells
+
+    def delete_flow_cell_data(self, parsed_flow_cell: FlowCellDirectoryData) -> None:
```

Comment:
Maybe you only need to give the id as an argument, not the complete flow cell
```suggestion
    def delete_flow_cell_data(self, parsed_flow_cell_id: str) -> None:
```